src/zkevm_specs/evm/table.py

```diff
@@ -202,6 +219,12 @@ class Tables:
         ]
     )
 
+    # Each row in BlockTable contains:
+    # - tag
+    # - block_number_or_zero
```

Comment:
block number is zero seems make no sense as it begins from 1.

Reply:
This is only for `BlockContextFieldTag.BlockHash` usage, I have added some more comments in [`c52c2f1`](https://github.com/appliedzkp/zkevm-specs/pull/77/commits/c52c2f16158aaa629906a38ff5d0448b3b115a76)